scripts/generate-manifest.js

```diff
@@ -128,3 +139,49 @@ function manifestLayerV2(data, hostname) {
   }
   return layer;
 }
+
+function manifestLayerV6(data, hostname, opts) {
+  const fields = data.fieldMapping.map(fieldMap => ({
+    type: fieldMap.type,
+    id: fieldMap.name,
+    label: { ...{ en: fieldMap.desc }, ...getFieldLabels(fieldMap.name, opts.fieldInfo) },
+  }));
+  const layer = {
+    layer_id: data.name,
+    created_at: data.createdAt,
+    attribution: data.attribution,
+    formats: data.emsFormats.map(format => {
+      return {
+        format: format.type,
+        url: `https://${hostname}/files/${format.file}?elastic_tile_service_tos=agree`,
+        legacy_default: format.default || false,
+      };
+    }),
+    fields: fields,
+    layer_name: data.humanReadableName,
+  };
+  return layer;
+}
+
+function getDefaultFormat(emsFormats) {
+  return emsFormats.filter(format => format.default);
+}
+
+function getFieldLabels(fieldName, fieldInfo) {
+  if (fieldName.startsWith('label_') && _.get(fieldInfo, 'name.i18n')) {
+    const lang = fieldName.replace('label_', '');
+    const labels = _.mapValues(fieldInfo.name.i18n, (label) => {
+      return `${label} (${lang})`;
+    });
+    return labels;
+  } else if (_.get(fieldInfo, `${fieldName}.i18n`)) {
+    return fieldInfo[fieldName].i18n;
+  } else return;
```

Comment:
Easy to miss that return
```suggestion
  } else {
    return;
  }
```